app/views/foods/create.html.erb

```diff
@@ -0,0 +1,4 @@
+<div>
+  <h1 class="font-bold text-4xl">Foods#create</h1>
+  <p>Find me in app/views/foods/create.html.erb</p>
+</div>
```

Comment:
The destroy and create methods don't require any views.

Reply:
Done! Also, I had already handled routing when logged out just before you submitted your pr. I also added data into the seed file, but note that this task wasn't within the required requirements for this issue.